docs/README.md

```diff
@@ -0,0 +1,56 @@
+###Reproducing
+
+Get the files from the appropriate bucket & RUN_CONFIG
+```
+wget --no-parent https://awi-ciroh-ngen-data.s3.us-east-2.amazonaws.com/AWI_001/AWI_03W_113060_001.tar.gz .
+
+tar -xvf AWI_03W_113060_001.tar.gz
+```
+Then we can confirm file location and integrity against the example JSON file.
```

Comment:
We discussed splitting the tar into several pieces. This will work for tomorrow. If we can set up the split files, that would be more consistent with the eventual goal.

Reply:
It's a stopgap for sure, I can add a TODO there if you like